### ISSUE TITLE
Update MegaFlashROM related ROMs booting procedure

### ISSUE THREAD

**Issue by Steveneska** [Konamiman/Nextor#126]
I'm not sure if this is the correct name for this enhancement but I've noticed that while the Sunrise ROMs with device driver 0.1.7 could boot in MODE 80 and apply the correct COLOR parameters as saved by the user, the MegaFlashROM related ROMs would ignore the user saved settings and still boot in MODE 40 with the computer default COLOR parameters. Could they be updated to reflect the same behavior as the Sunrise IDE ROMs with device driver 0.1.7?

**Comment by Braniegk** [comment]
That would be nice... if Konamiman has enought time...

**Comment by Steveneska** [comment]
I could submit a PR if I could get Nextor to compile but I can't get it to work with WSL.

**Comment by Konamiman** [comment]
@Steveneska What problem are you having with the build? Is it the same as https://github.com/Konamiman/Nextor/issues/127? If we can sort out that issue then yes, please feel free to submit a pull request.

**Comment by Steveneska** [comment]
No. I followed the build instructions, but it wasn't finding the tools (N80, L80...) in PATH even though I installed all the required tools and put them in the bin folder and made sure the bin folder was actually in PATH

**Comment by Konamiman** [comment]
@Steveneska Can you please try running the following? (replacing `/usr/bin` with the real path where you have put the tools)

```
N80=/usr/bin/N80 LK80=/usr/bin/LK80 LB80=/usr/bin/LB80 make
```

The makefile script will take the path of the N80, LK80 and LB80 tools from the environment variables of the same name if they exist, so if it's only a "tools not found" issue this should work for you.

**Comment by Steveneska** [comment]
@Konamiman I did as you just instructed and still get the same error as previously:

```
make -C kernel
make[1]: Entering directory '/home/steveneska/Nextor/source/kernel'
Makefile:75: *** "ERROR: can't execute /usr/bin/N80, is it installed/in PATH?".  Stop.
make[1]: Leaving directory '/home/steveneska/Nextor/source/kernel'
make: *** [Makefile:7: all] Error 2
```

I've tried with both flavors (framework dependent and self-contained) of N80, LIB80 and LK80 and got the same result

**Comment by Konamiman** [comment]
Hi @Steveneska, could you please run the following command in your WSL console and paste the output here?

`ls -l /usr/bin/N80`

**Comment by Steveneska** [comment]
@Konamiman This is what I get: `-rw-r--r-- 1 steveneska steveneska 64522514 Mar 20  2023 /usr/bin/N80`

**Comment by Konamiman** [comment]
@Steveneska Mystery solved then: your N80 file isn't set as executable. You need to run the following:

`sudo chmod +x /usr/bin/N80`

and the same with LK80 and LB80. Then repeating the `ls` command will give you `-rwx...` (the "x" means it's executable) and it will work.

**Comment by Steveneska** [comment]
Yeah I get this now: `-rwxr-xr-x 1 steveneska steveneska 64522514 Mar 20  2023 /usr/bin/N80`

Thank you for helping troubleshoot this and really sorry for the trouble, I'm very new to Linux and didn't know I could do this.

**Comment by Steveneska** [comment]
I will try to compile my changes, test to see if they check out and report back to you

**Comment by Steveneska** [comment]
Ran through this error while compiling Nextor with no modifications

```
sdasz80 -o bank5/fdisk_crt0.rel bank5/fdisk_crt0.s
sdcc -o bank5/ --code-loc 0x4120 --data-loc 0x8020 -mz80 --disable-warning 196 --disable-warning 84 --disable-warning 85 \
     --max-allocs-per-node 1000 --allow-unsafe-read --opt-code-size --no-std-crt0 bank5/fdisk_crt0.rel bank5/fdisk.c
bank5/../../tools/C/asmcall.h:60: syntax error: token -> '__sdcccall' ; column 147
bank5/../../tools/C/asmcall.c:19: error 65: function 'AsmCallAlt' already has body
bank5/../../tools/C/asmcall.c:19: syntax error: token -> '__sdcccall' ; column 155
make: *** [Makefile:554: bank5/fdisk.dat] Error 1
```

**Comment by Konamiman** [comment]
@Steveneska Which version of sdcc are you using? Yoy need v4.2 or newer.

**Comment by Steveneska** [comment]
@Konamiman I'm using version 4.0.0+dfsg-2. Says it's the newest

Edit: Had to upgrade Ubuntu. Compiling works now.

**Comment by Konamiman** [comment]
Implemented in https://github.com/Konamiman/Nextor/pull/132
